server/activity.go

```diff
@@ -67,6 +67,16 @@ func (s *Server) registerActivityRoutes(g *echo.Group) {
 	g.GET("/activity", func(c echo.Context) error {
 		ctx := context.Background()
 		activityFind := &api.ActivityFind{}
+		if creatorIDStr := c.QueryParams().Get("creator"); creatorIDStr != "" {
```

Comment:
creator -> user.

creator is an implementation detail. What we want to do is fetching all activities related to this user. it's also consistent with other places.